src/sources/metadatasource.h

```diff
@@ -20,7 +20,10 @@ class MetadataSource {
   public:
     virtual ~MetadataSource() = default;
 
-    static QDateTime getFileSynchronizedAt(const QFileInfo& fileInfo);
+    /// Get the synchronization time of the given file.
```

Comment:
This comment is wrong. the function returns the file modification time, from the HDD, this is not related to any synchronization inside Mixxx.

Reply:
getFileSynchronizedAt 12:00 reads as if the file has been synchonized at 12:00. But this is not the case, an external program has touched the file at 12:00 and the file is no longer synchonized.

Reply:
> getFileSynchronizedAt 12:00 reads as if the file has been synchonized at 12:00. But this is not the case, an external program has touched the file at 12:00 and the file is no longer synchonized.

I kind of agree that the naming may be confusing, however the method name was not changed in this PR. The PR is supposed to fix a bug, and think it's unreasonable to request refactoring larger parts of the code in a bugfix PR.

Reply:
Thank you, Jan.

Reply:
OK